Makefile

```
@@ -304,7 +304,7 @@ buildah-dev-build:
 
 .PHONY: podman-dev-build
 podman-dev-build:
-	podman build --tag ${IMG} -f ./Dockerfile .
+	podman build --tag ${IMG} --volume $(go env GOCACHE):/go/.cache:z -f ./Dockerfile .
```

I would like to suggest adding a variable GOCACHE ?= $(shell go env GOCACHE) near the top of the makefile and then having the following here:

```suggestion
	podman build --tag ${IMG} --volume "${GOCACHE}:/go/.cache:z" -f ./Dockerfile .
```

---

Yes. I'd like to make it even smarter: if `go env GOCACHE` is unset, don't even include the `--volume` arg.

I'll hack at this next week.

---

Done, ptal.